README.md

```diff
@@ -0,0 +1,3 @@
+
+
+Document Management System provides a restful API for users to create and manage documents giving different privileges based on user roles and managing authentication using JWT. The API has routes, each dedicated to a single task that uses HTTP response codes to indicate API status and errors.
```

Comment:
You can expand on your readme telling us your app dependencies, how to run your app, how to test it, limitations of the app, urls, endpoints (for the api part) and any other useful information.